src/analysis/souffle/VisionPipelineAuthLogicOutput.dl

```diff
@@ -0,0 +1,26 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-----------------------------------------------------------------------------
+#include "authorization_logic.dl"
+
+// This file is roughly what should be produced by authorization logic as an 
```

Comment:
Is this one of the expected files? If so, shouldn't it move to somewhere under examples or test directory?